src/metabase/driver/sql/query_processor.clj

```diff
@@ -153,6 +153,12 @@
 (defmethod date [:sql :year-of-era]      [_driver _ expr] (hx/year expr))
 (defmethod date [:sql :week-of-year-iso] [_driver _ expr] (hx/week expr))
 
+(defmulti datetime-diff
+  "Return a HoneySQL form for calculating the datetime-diff for a given unit."
+  {:arglists '([driver unit field-or-value field-or-value])}
```

Comment:
```suggestion
  {:arglists '([driver unit field-or-value field-or-value]), :added "0.46.0"}
```

Since third-party driver authors are implementing Metabase drivers it's nice to add a little bit of extra metadata like this so driver authors can know when something was added just in case they want to support multiple versions in one driver

Reply:
You should also add a little blurb about this new method and the purpose of it in 

https://github.com/metabase/metabase/blob/master/docs/developers-guide/driver-changelog.md

Reply:
Cool, updated the changelog https://github.com/metabase/metabase/commit/2e5f76df104cddddc8297b9121df4675f646100b